code/data_setup/parse_imagenet_bb.py

```diff
@@ -38,21 +39,22 @@ def parse_file(bbox_dir, filename):
             bounding_box = child.find('bndbox')
             xmin, xmax, ymin, ymax = parse_bb(bounding_box)
 
-            object_lst = [width, height, name, subcategory, xmin, 
+            object_lst = [filename, width, height, name, subcategory, xmin,
```

Comment:
Nice, excited to dig in! I think I should have some time to play around with this this morning. 

I'm good with the `x0, y0, x1, y1` convention, so why don't we move forward with that. The other convention was what I think was used in the bbox files for imagenet, so I had just used that. No particular reason/preference for it.